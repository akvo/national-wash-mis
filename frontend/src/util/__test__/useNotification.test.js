import { useNotification } from "../hooks";
import { render } from "@testing-library/react";
import { useEffect } from "react";

const TestNotification = () => {
  const { notify } = useNotification();

  useEffect(() => {
    notify({
      type: "error",
      message: "Not Found",
    });
  }, [notify]);
  return <div>Message: Not Found</div>;
};

jest.mock("antd");

describe("util/hooks", () => {
  test("test useNotification", () => {
    const { asFragment } = render(<TestNotification />);
    expect(asFragment()).toMatchSnapshot();
  });
});
