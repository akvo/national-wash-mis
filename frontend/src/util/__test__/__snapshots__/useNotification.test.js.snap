// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util/hooks test useNotification 1`] = `
<DocumentFragment>
  <div>
    Message: Not Found
  </div>
</DocumentFragment>
`;
